src/QsCompiler/Compiler/CompilationLoader.cs

```diff
@@ -568,6 +562,12 @@ public CompilationLoader(SourceLoader loadSources, ReferenceLoader loadReference
                 steps.Add((rewriteStep.Priority, () => this.ExecuteAsAtomicTransformation(rewriteStep, ref this.compilationStatus.PreEvaluation)));
             }
 
+            if (this.config.IsExecutable && !this.config.SkipMonomorphization)

```

Comment:
Was this moved after `GenerateFunctorSupport` for some reason?

Reply:
Yes Monomorphization is going to be the last rewrite step processed before external rewrite steps now. I guess the order in this file is not super important, as the priority number is actually the thing that determines the order, but having the order here match that I think makes things more consistent.

Reply:
The reason Monomorphization has to be processed after functor support is that we don't have a way to deal with auto-generated functor specializations that have not been generated yet when building the concrete call graph, so the solution is to run functor generation first to make sure all of the functor specializations are generated and present.

Reply:
Thanks for the explanation Scott! Yes, I like the consistency.